sympy/physics/control/control_plots.py

```diff
@@ -882,6 +919,10 @@ def bode_plot(system, initial_exp=-5, final_exp=5,
         If ``True``, the plot will have a grid. Defaults to True.
     show_axes : boolean, optional
         If ``True``, the coordinate axes will be shown. Defaults to False.
+    freq_unit : string, optional
+        User can choose between 'rad/sec' (radians/second) and 'Hz' (Hertz) as frequency units.
+    phase_unit : string, optional
+        User can choose between 'rad' (radians) and 'deg' (degree) as phase units.
```

Comment:
And here

Reply:
Thanks @oscargus .I have made all suggested changes ,you can have a look now !